test/object_store/test_backup.py

```diff
@@ -200,8 +200,7 @@ async def test_backup_is_abortable_in_s3_client(manager: ManagerClient, s3_serve
     '''check that backing up a snapshot for a keyspace works'''
     await do_test_backup_abort(manager, s3_server, breakpoint_name = "backup_task_pre_upload", min_files = 0, max_files = 1)
 
-@pytest.mark.asyncio
-async def test_simple_backup_and_restore(manager: ManagerClient, s3_server):
+async def do_test_simple_backup_and_restore(manager: ManagerClient, s3_server, do_abort = False):
```

Comment:
nit, could also parameterize the test, like
```py
@pytest.mark.parametrize("do_abort", [True, False])
aync def test_simple_backup_and_restore(manager: ManagerClient, s3_server, do_abort):
   # ...
```

Reply:
True. I just find it easier to navigate when tests are named proxies instead.